VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -154,11 +154,11 @@ private void setCurrentUnit(CourseComponent component){
 
         environment.getDatabase().updateAccess(null, selectedUnit.getId(), true);
 
-        CourseComponent parent = component.getParent();
+        CourseComponent parentSectionComponent = component.getParent().getParent();
         String prefName = PrefManager.getPrefNameForLastAccessedBy(getProfile()
             .username, selectedUnit.getCourseId());
         final PrefManager prefManager = new PrefManager(MainApplication.instance(), prefName);
-        prefManager.putLastAccessedSubsection(parent.getId(), false);
+        prefManager.putLastAccessedSubsection(parentSectionComponent.getId(), false);
```

Comment:
If we are only storing the course section (not subsection) as the last accessed item, then we might as well remove this as that has already been implemented now in `CourseOutlineActivity`. Note that the subsection is stored in some other areas as well, and they should all be removed and/or modified.